src/request/create/CreateApi.js

```diff
@@ -1,12 +1,19 @@
 /* global TopLevelApi */
 /* global Create */
 /* global Nimiq */
-
+/* global Errors */
 class CreateApi extends TopLevelApi { // eslint-disable-line no-unused-vars
     /**
      * @param {KeyguardRequest.CreateRequest} request
      */
     async onRequest(request) {
+        /** @type {HTMLElement} */
+        const $appName = (document.querySelector('#app-name'));
+        $appName.textContent = request.appName;
+        /** @type {HTMLButtonElement} */
+        const $cancelLink = ($appName.parentNode);
+        $cancelLink.classList.remove('display-none');
+        $cancelLink.addEventListener('click', () => this.reject(new Errors.Cancel()));
         const parsedRequest = CreateApi._parseRequest(request);
```

Comment:
I'd say to move the cancel button init to the end of the function (potentially before `run()`, because errors in `_parseRequest` would abort the request anyway.